quickjs.c

```diff
@@ -62,7 +62,7 @@
 #define NO_TM_GMTOFF
 #endif
 
-#if !defined(EMSCRIPTEN) && !defined(__wasi__) && !defined(__STDC_NO_ATOMICS__)
+#if !defined(EMSCRIPTEN) && !defined(__wasi__) && !__STDC_NO_ATOMICS__
```

Comment:
We'd still need this change I think.

Reply:
oh yea, that's right

Reply:
Can I make a small PR for that or fix this one so you could merge it? @saghul

Reply:
Sure. Let's add the CI too I guess. Just build the qjs target so the test runner doesn't get built.

Reply:
Ok so I will fix this one

Reply:
I will delete all the atomics things I tried in `quickjs-c-atomics.h` as they don't work, ok?

Reply:
Sure thing!

Reply:
Seems like it's greening, is that exactly what you wanted? If not I could fix it. Nevertheless big thanks for all !

Reply:
Yep, this is it!